### PR TITLE
Added --data-raw support

### PR DESCRIPTION
**Pull request by mkp95**
Raw data support

### CODE REVIEW

**Review by ScotchE — APPROVED** [review]

**Comment by incarnate** [comment]
Hi @mkp95 - happy new year and thanks for the pull request!

Can you please narrow the changes to just the support for `--data-raw` - I'm uncomfortable with disabling HTTPS verification as a default. See #5 for more discussion on that.

**Comment by NabiKAZ** [comment]
@mkp95 
@incarnate said correct, please remove detect https from your code. I need just data-row support for now.
It's been about 3 months; Can you do that or think of something else?

**Comment by mkp95** [comment]
> @mkp95
> @incarnate said correct, please remove detect https from your code. I need just data-row support for now.
> It's been about 3 months; Can you do that or think of something else?

I think I have to edit the whole file again will do it by 1st April..

**Comment by mkp95** [comment]
now you can approve the pull request .. i removed all alterations and added data raw support only

File: resources/js/curl-to-php.js [hunk 1]
```diff
@@ -261,10 +261,10 @@ function curlToPHP(curl) {
 			loadData(cmd.d);
 		if (cmd.data)
 			loadData(cmd.data);
+		if (cmd['data-raw'])
+			loadData(cmd['data-raw']);
```
Comment (mkp95) [inline resources/js/curl-to-php.js:265]: data raw added

**Comment by NabiKAZ** [comment]
@incarnate

**Comment by incarnate** [comment]
Apologies for the slow merge, thanks for the contribution!

**Pull request merged**